### ISSUE TITLE
Add gpg verification support 

### ISSUE THREAD

**Issue by JensHeinrich** [ansible-collections/community.crypto#199]
##### SUMMARY

Add a `gpg_verify` module

##### ISSUE TYPE
- Feature Idea

##### ADDITIONAL INFORMATION

This probably could be done using the python-gnupg library.
I would like to implement it TBH, but not if it is not deemed useful.

```yaml

```

**Comment by felixfontein** [comment]
This definitely sounds interesting. I don't know python-gnupg though, and have no idea how common that library is. Direct interaction with `gpg` is not that much fun in most cases, though, and verification seems to be one of them, so I don't think we have many choices :)

**Comment by JensHeinrich** [comment]
I can confirm that it is hard, as it does (by default at least) put user messages on STDERR causing tasks to fail

**Comment by felixfontein** [comment]
Related issue: #238

**Comment by felixfontein** [comment]
@JensHeinrich are you still interested in contributing such a module?

**Comment by JensHeinrich** [comment]
Yes, I think I started writing something already, but need to find it again
